core/build.gradle.kts

```diff
@@ -47,8 +47,26 @@ android {
     buildConfigField("long", "VERSION_CODE", "${generateVersionCode()}")
   }
   buildTypes {
+    getByName("debug") {
+      dependencies {
+        implementation(Libs.objectbox_kotlin)
+        implementation(Libs.objectbox_rxjava)
+      }
+    }
     getByName("release") {
       isMinifyEnabled = false
+      apply(plugin = "io.objectbox")
```

Comment:
why we are adding this code here? we already declared this here https://github.com/kiwix/kiwix-android/blob/563170cd76c9b6f2e8a0562dbefd89850e551249/core/build.gradle.kts#L19